pkg/proxy/apis/config/validation/validation_test.go

```diff
@@ -252,10 +271,10 @@ func TestValidateKubeProxyConfiguration(t *testing.T) {
 		},
 		{
 			config: kubeproxyconfig.KubeProxyConfiguration{
-				BindAddress:        "192.168.59.103",
+				BindAddress:        []string{"192.168.59.103"},
 				HealthzBindAddress: "0.0.0.0:10256",
 				MetricsBindAddress: "127.0.0.1:10249",
-				ClusterCIDR:        "192.168.59.0/24",
+				ClusterCIDR:        []string{"192.168.59.0/24"},
```

Comment:
maybe we should add some validation with IPv6 and dual-stack cidrs

Reply:
I stuck in a case, I'll put in more later.